src/Microsoft.AspNetCore.Routing/RouteOptions.cs

```diff
@@ -10,10 +10,15 @@ namespace Microsoft.AspNetCore.Routing
     public class RouteOptions
     {
         /// <summary>
-        /// Gets or sets a value indicating whether all generated URLs are lower-case.
+        /// Gets or sets a value indicating whether all generated URLs are lower-case. it doesn't include the query string.
         /// </summary>
         public bool LowercaseUrls { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether all generated QUERY STRINGS are lower-case. property will when LowercaseUrls true.
+        /// </summary>
+        public bool LowercaseQueryStrings { get; set; }
```

Comment:
Suggesting for this:
`Gets or sets a value indicating whether a generated query strings are lower-case. This property will be unless <see cref="LowercaseUrls" /> is also <c>true</c>`